packages/x-charts/src/LineChart/MarkElement.tsx

```diff
@@ -100,6 +100,9 @@ function MarkElement(props: MarkElementProps) {
       onClick={onClick}
       cursor={onClick ? 'pointer' : 'unset'}
       {...interactionProps}
+      data-series={id}
+      data-highlighted={isHighlighted || undefined}
+      data-faded={isFaded || undefined}
```

Comment:
Thos attributes should be added either in the mark or the circle, but probably not both